ui/src/app/components/cases/test-case-details.component.ts

```diff
@@ -227,7 +227,12 @@ export class TestCaseDetailsComponent extends BaseComponent implements OnInit {
     testCases = new InfiniteScrollableDataSource(this.testSuiteService, ",testcaseId:" + this.testCaseId);
     waitTillRequestResponds();
     let _this = this;
-
+    /**
+     * This is a function which is used to add a timeout when testCases.isFetching returns true. Gives a timeout for the testcases to be fetched.
+     * @example
+     *
+     * waitTillRequestResponds()
+     * */
     function waitTillRequestResponds() {
```

Comment:
seems like duplicate handler, plz try to optmize/reuse

Reply:
that function is declared only once in this file.